jmclient/jmclient/__init__.py

```diff
@@ -53,7 +53,8 @@
     wallet_tool_main, wallet_generate_recover_bip39, open_wallet,
     open_test_wallet_maybe, create_wallet, get_wallet_cls, get_wallet_path,
     wallet_display, get_utxos_enabled_disabled, wallet_gettimelockaddress,
-    wallet_change_passphrase)
+    wallet_change_passphrase, bip32pathparse, WalletView, WalletViewAccount,
+    WalletViewBranch, WalletViewEntry)
```

Comment:
I haven't usually thought much about this, but there may be logic in distinguishing this kind of import:

```
from jmclient import Blah
```

from this kind of import:

```
from jmclient.wallet_utils import Blah
```

.. although I've never been too clear on the value of the distinction; the latter might make sense for cases like this, because: the internal `WalletView*` objects are supposed to be contained by `WalletView` itself, and so shouldn't be needed by calling code (probably; but let's say, for the purpose of this discussion). However it's clearly needed by test code to test the individual components, so maybe specifically in tests we use the `from jmclient.wallet_utils import ...` syntax?

Reply:
You're right, didn't come into my mind when creating this.